<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>560</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <widget class="QScrollBar" name="vscrlStrips">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>69</y>
      <width>16</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>91</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblFrames">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>64</width>
      <height>13</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>13</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>47</width>
      <height>13</height>
     </size>
    </property>
    <property name="text">
     <string>Frames:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblAnims">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>64</width>
      <height>13</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>13</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>47</width>
      <height>13</height>
     </size>
    </property>
    <property name="text">
     <string>Animations:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnimFrameNext">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnimPause">
    <property name="geometry">
     <rect>
      <x>356</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>||</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrlAnimFrameBar">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>9</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnimPlay">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnimFramePrev">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="cFrameBar" name="fieldAnimFrameBar">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>31</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="cPreviewArea" name="fieldAnimPreview">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>195</y>
      <width>581</width>
      <height>338</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>312</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="editAnimSpeed">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>167</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomMinus">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomPlus">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>166</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_StripAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_StripDel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_StripGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="stripList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="indentation">
     <number>8</number>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="allColumnsShowFocus">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>147</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>147</number>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
   </widget>
   <widget class="cFrameGrid" name="frameGrid">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>70</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DropOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>12</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>24</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>16</number>
    </attribute>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QListWidget" name="stripFrameList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>172</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cFrameBar</class>
   <extends>QFrame</extends>
   <header>cframebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cPreviewArea</class>
   <extends>QFrame</extends>
   <header>cpreviewarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cFrameGrid</class>
   <extends>QTableWidget</extends>
   <header>cframegrid.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
